ROADMAP.md

```diff
@@ -25,16 +26,6 @@ For a more detailed view see https://github.com/orgs/libp2p/projects/4/views/1.
 
 <!-- markdown-toc end -->
 
-## TLS support
-
-| Category     | Status      | Target Completion | Tracking                                        | Dependencies                       | Dependents |
-|--------------|-------------|-------------------|-------------------------------------------------|------------------------------------|------------|
-| Connectivity | In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/pull/2945 | [QUIC](#experimental-quic-support) |            |
-
-
-This allows us to secure both TCP and QUIC connections using TLS. This is a requirement for QUIC
-support. Running TLS on top of TCP is a nice to have, since we already have noise.
-
 ## Experimental QUIC support
```

Comment:
Should we still label it as _Experimental_ if the item goes beyond the first implementation?

Reply:
:+1: 90e503fb8